src/auth/auth.utils.ts

```diff
@@ -0,0 +1,3 @@
+export function isStringValue(value: string): value is `${number}${"s" | "m" | "h" | "d" | "w" | "y"}` {
```

Comment:
OK, definitely declare a type alias because this is prone to bugs in the long run